unlock-app/src/components/interface/locks/Manage/index.tsx

```diff
@@ -54,6 +56,13 @@ export function downloadAsCSV(cols: string[], metadata: any[]) {
 }
 
 const ActionBar = ({ lockAddress, network }: ActionBarProps) => {
+  const { isLoading: isLoadingMetadata, data: metadata } = useMetadata({
+    lockAddress,
+    network,
+  })
+
+  const { isEvent } = getLockTypeByMetadata(metadata)
```

Comment:
Does it work when metadata is undefined without error?

Reply:
tested and yes it is fine!